### ISSUE TITLE
google provider cannot configure Cloud Operations for GKE to set logging and monitoring type

### ISSUE THREAD

**Issue by sharkymcdongles** [hashicorp/terraform-provider-google#8753]
![image](https://user-images.githubusercontent.com/29139537/112196686-90fd9980-8c0b-11eb-9728-dfcd15e44b75.png)

This is on the latest provider version.

There is logging service, but you can't set all these settings and naming is confusing from what Google shows in GUI and via CLI:

https://registry.terraform.io/providers/hashicorp/google-beta/latest/docs/resources/container_cluster#logging_service

**Comment by venkykuberan** [comment]
@sharkymcdongles  Provider schema attributes are mapped to API attributes. The GUI setting you see is a combination of two API attributes `loggingService & monitoring Service` refer the [api doc](https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1beta1/projects.locations.clusters).

In terraform config you have to use `logging_service` & `monitoring_service` fields for the same. Hope that helps

**Comment by sharkymcdongles** [comment]
@venkykuberan I tested to see if these settings are the same, but it seems not or there is another setting elsewhere. I provisioned a cluster with the standard "system and workload logging and monitoring" setting. I then used the CLI to describe the cluster both with beta and regular describe cluster commands. This resulted with:

loggingService: logging.googleapis.com/kubernetes
monitoringService: monitoring.googleapis.com/kubernetes


I then changed the setting to "System logging and monitoring only". I then described with the CLI and the result was the same:

loggingService: logging.googleapis.com/kubernetes
monitoringService: monitoring.googleapis.com/kubernetes

I guess this means they haven't exposed this setting via their API yet? I see no other setting listed on the API page. It does appear settable via the CLI tool though: https://cloud.google.com/sdk/gcloud/reference/beta/container/clusters/create#--enable-logging-monitoring-system-only

**Comment by venkykuberan** [comment]
It seems like you are looking for `Cluster Telemetry` attribute - https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1beta1/projects.locations.clusters#Cluster.ClusterTelemetry. 

**Provider doc** 

The cluster_telemetry block supports
```
type - Telemetry integration for the cluster. Supported values (ENABLED, DISABLED, SYSTEM_ONLY); SYSTEM_ONLY (Only system components are monitored and logged) is only available in GKE versions 1.15 and later.
```

**Comment by sharkymcdongles** [comment]
> It seems like you are looking for `Cluster Telemetry` attribute - https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1beta1/projects.locations.clusters#Cluster.ClusterTelemetry.
> 
> **Provider doc**
> 
> The cluster_telemetry block supports
> 
> ```
> type - Telemetry integration for the cluster. Supported values (ENABLED, DISABLED, SYSTEM_ONLY); SYSTEM_ONLY (Only system components are monitored and logged) is only available in GKE versions 1.15 and later.
> ```

Seems this is the setting I couldn't find thanks! Strangely when I changed it by hand to the above setting then rerun my terraform apply action, it didn't detect or reset the value to the other despite me having no lifecycle ignore changes on the clust resource. Either way I think it is fine now and will close this. thanks again for the help.
